pkg/registry/styles/projects.less

```diff
@@ -1,6 +1,9 @@
 
 @tag-border: @link-color;
 
+#add-entity {
```

Comment:
It's illegal HTML to have duplicate ids on the same page. But I'll take a look at how to solve this better.